rocket/models.py

```diff
@@ -1,14 +1,16 @@
 from __future__ import unicode_literals
-
+from django.contrib.auth.models import User
 from django.db import models
 from django.utils import timezone
 from django.forms import ModelForm
 from django import forms
 
 
+
+
 # Create your models here.
```

Comment:
Redundant addition of new lines. Can you make sure from now onwards that you don't add unwanted changes. Try doing `git add --patch`.